rllib/core/rl_trainer/trainer_runner.py

```diff
@@ -42,80 +42,84 @@ def __init__(
         self,
         trainer_class: Type[RLTrainer],
         trainer_config: Mapping[str, Any],
-        compute_config: Mapping[str, Any],
+        num_gpus: int,
```

Comment:
having compute_config was I think a good choice. later we can convert it to a self contained dataclass that prompts the user what the allowed parameters are.

Reply:
I am using that comput_config assumption in the trainer_config_runner.